src/methods/tx.rs

```diff
@@ -1,3 +1,29 @@
+//! This module allows you to query the status of a transaction signed by an account_id
+//!
+//! ## Example
+//!
+//! ```
+//! use near_jsonrpc_client::{methods, JsonRpcClient};
+//! use near_jsonrpc_primitives::types::transactions::TransactionInfo;
+//!
+//! # #[tokio::main]
+//! # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//! let client = JsonRpcClient::connect("https://archival-rpc.mainnet.near.org");
+//! let tx_hash = "B9aypWiMuiWR5kqzewL9eC96uZWA3qCMhLe67eBMWacq".parse()?;
+//!
+//! let request = methods::tx::RpcTransactionStatusRequest {
+//!     transaction_info: TransactionInfo::TransactionId {
```

Comment:
^^^ following the above comment

```suggestion
//!     transaction_info: methods::tx::TransactionInfo::TransactionId {
```